kpops/components/base_components/base_defaults_component.py

```diff
@@ -122,25 +122,23 @@ def extend_with_defaults(cls, config: KpopsConfig, **kwargs: Any) -> dict[str, A
             elif is_dataclass(v):
                 kwargs[k] = asdict(v)
 
-        log.debug(
-            typer.style(
-                "Enriching component of type ", fg=typer.colors.GREEN, bold=False
-            )
-            + typer.style(cls.type, fg=typer.colors.GREEN, bold=True, underline=True)
-        )
         main_default_file_path, environment_default_file_path = get_defaults_file_paths(
             config, ENV.get("environment")
         )
         defaults = cls.load_defaults(
             main_default_file_path, environment_default_file_path
         )
+        log.debug(
```

Comment:
why did you move the log position? afaict it was correct before

Reply:
I moved it because it was before the method `get_defaults_file_paths` which returns the default path and does not do any "enriching". The enriching is happening in the `update_nested_pair`. Therefore I moved it right before the method call